WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -430,9 +430,32 @@ public void onCreate(Bundle savedInstanceState) {
             initSubFilterViewModel();
         }
 
-        handleReblogStateChanges();
+        mViewModel.getNavigationEvents().observe(getViewLifecycleOwner(),
```

Comment:
This method is almost identical to `handleReblogStateChanges` which is removed a few lines below.